pkg/reconciler/dependencybuild/buildrecipeyaml.go

```diff
@@ -607,6 +608,10 @@ func memoryLimits(jbsConfig *v1alpha1.JBSConfig, additionalMemory int) (*memLimi
 	if err != nil {
 		return nil, err
 	}
+	limits.buildLimitCPU, err = resource.ParseQuantity(settingOrDefault(jbsConfig.Spec.BuildSettings.BuildLimitCPU, "2"))
```

Comment:
Was this meant to be different to what is configured in the dev-template/config.yaml?

Reply:
I don't know what is a reasonable default, I just went a bit more conservative than the dev script.